test/Microsoft.AspNet.Mvc.Core.Test/Infrastructure/ObjectResultExecutorTest.cs

```diff
@@ -0,0 +1,499 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
```

Comment:
These were ported and then gutted. There was a lot of duplication and unnecessary complexity in these tests.